### ISSUE TITLE
Building Windows-XP compatible dlls with modern Visual Studio versions

### ISSUE THREAD

**Issue by FreeSlave** [FWGS/hlsdk-portable#281]
@a1batross said it's possible to keep XP compatibility in modern Visual Studio versions, referencing these lines from xash3d's build scripts:
https://github.com/FWGS/xash3d-fwgs/blob/3ad60a0fa1af834b13a73051bf5cd535ee83b74a/wscript#L142 and https://github.com/FWGS/xash3d-fwgs/blob/3ad60a0fa1af834b13a73051bf5cd535ee83b74a/scripts/waifulib/compiler_optimizations.py#L54

**Comment by a1batross** [comment]
Yes, just don't include any STL headers. These are probably totally broken.
